spec/core_functions/color/mix/error.hrx

```diff
@@ -60,6 +60,8 @@ Error: $color1: To use color.mix() with non-legacy color lch(20% -20 0deg), you
 <===> null_method/non_legacy/color2/options.yml
 :ignore_for:
   - libsass
+:todo:
+  - dart-sass
```

Comment:
I think you have noted this in the wip dart sass [PR](https://github.com/sass/dart-sass/pull/1845/files). I marked this one as to do as well.